docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+Policy as code with provisioning tools like Terraform merges infrastructure-as-code and compliance-as-code into a single workflow. The Chef InSpec can be used with *Terraform* in two different ways to confirm compliance:
+
+- Audit provisioned infrastructure: When developing Terraform code for repositories, Chef InSpec can verify the resources are updated to match the tested and approved criteria.
+
+- Terraform code declaration: Chef InSpec declares the infrastructure configuration in the test-driven development, and Terraform can be used to provision resources accordingly. In this way, Terraform manages provisioning while InSpec ensures the provisioned resources meet the organization’s policy requirements.
+
+Together, these capabilities ensure that with Chef, you have a single way to detect, correct, and automate any issue in any cloud-based environment.
+
+### Docker Benchmark Recommendations
+
+The following Docker benchmark recommendations apply to both the host and the Docker components:
+
+- Host configuration: Secure the host on which the Docker engine runs, and the container it hosts is safe.
+- Docker daemon configurations: Secure the Docker Daemon behavior that manages all containers on the Docker host.
+- Container images and build files: Ensure trusted images and verified packages.
+- Container runtime: Ensure no compromise is made on the complaint container startup (host) and runtime parameter configurations, such as avoiding privileged containers and SSH usage within containers.
+- Docker security operations: Ensure a limit on the number of containers and respective images on the same host.
+- Docker swarm configuration: Secure Docker Swarm, the container orchestrator that can manage container clusters and their lifecycle.
+
+### CIS Benchmark Recommendations
+
+The CIS benchmark for Kubernetes includes configuration checks on the following components:
+
+- Control Pane
+  - API server
+  - Controller manager
+  - Etcd
+  - Master node configuration
+  - Scheduler
+- Worker Nodes
+  - Kubelet configuration
+  - Worker node config files
+- Kubernetes Elements
+  - Container Network Interface (CNI) and secrets
+  - Network Policies
+  - Pods
+  - Role based access control
+
+### Docker and Kubernetes Validation
+
+Chef provides curated, codified, and ready-to-run CIS benchmark templates for Docker and Kubernetes clusters that performs security scans as an integrated part of automated DevSecOps workflows. By integrating compliance checks with automated workflow, you can validate that all your container and cloud-native environments are secure.
```

Comment:
Do the clusters perform security scans, or do the benchmark templates? It looks like the templates.

```suggestion
Chef provides curated, codified, and ready-to-run CIS benchmark templates for Docker and Kubernetes clusters. These CIS benchmark templates perform security scans as an integrated part of automated DevSecOps workflows. By integrating compliance checks with automated workflow, you can validate that all your container and cloud-native environments are secure.
```